pkg/prompt/prompt.go

```diff
@@ -75,6 +75,13 @@ func SysPrompt(ctx context.Context, envs []string, input string, _ chan<- string
 func sysPrompt(ctx context.Context, req types.Prompt) (_ string, err error) {
 	defer context2.GetPauseFuncFromCtx(ctx)()()
 
+	if req.Message != "" && len(req.Fields) == 1 && strings.TrimSpace(req.Fields[0]) == "" {
+		_, _ = fmt.Fprintln(os.Stderr, req.Message)
+		_, _ = fmt.Fprintln(os.Stderr, "Press enter to continue...")
+		_, _ = fmt.Fscanln(os.Stdin)
+		return "", nil
```

Comment:
Is it worth doing something like this?
```suggestion
		var errs []error
		 _, err := fmt.Fprintln(os.Stderr, req.Message)
		errs = append(errs, err)
		_, err = fmt.Fprintln(os.Stderr, "Press enter to continue...")
		errs = append(errs, err)
		_, err = fmt.Fscanln(os.Stdin)
		errs = append(errs, err)
		return "", errors.Join(errs...)
```

Reply:
I suppose so. I have never seen any of these fail, but it's worth reporting if they do